assets/queries/terraform/aws/alb_deletion_protection_disabled/test/positive.tf

```
@@ -0,0 +1,12 @@
+resource "aws_lb" "positive" {
```

name the file positive1.tf
```suggestion
resource "aws_lb" "positive1" {
```